arch/x86/core/intel64/locore.S

```diff
@@ -31,8 +31,16 @@ __start:
 1:
 #endif
 
+	/* FIXME: ...resetting TSC here is common to IA32 and Intel64... */
 	/* FIXME: ...disabling the PICs is common to IA32 and Intel64... */
 
+#ifdef CONFIG_BOOT_TIME_MEASUREMENT
+	xorl	%eax, %eax
+	xorl	%edx, %edx
+	movl	$X86_TIME_STAMP_COUNTER_MSR, %ecx
```

Comment:
indentation looks off here

Reply:
Whoops, I hastily airlifted this from the ia32 version. Fixed the spacing between the mnemonic and operands.